cosmovisor/process.go

```diff
@@ -83,7 +90,11 @@ func (l Launcher) Run(args []string, stdout, stderr io.Writer) (bool, error) {
 // It returns (false, nil) if the process exited normally without triggering an upgrade. This is very unlikely
 // to happened with "start" but may happened with short-lived commands like `gaiad export ...`
 func (l Launcher) WaitForUpgradeOrExit(cmd *exec.Cmd) (bool, error) {
-	currentUpgrade := l.cfg.UpgradeInfo()
+	currentUpgrade, err := l.cfg.UpgradeInfo()
+	if err != nil {
+		l.logger.Error().Err(err)
```

Comment:
Do we not want to return the error here?

Reply:
I am not sure, I have kept the same behavior, and before it was just logging the error: 
https://github.com/cosmos/cosmos-sdk/pull/11731/files#diff-dc70d911e1a6677efa35f14364247948c8e1d68a0e5f38c5345f4e33c485bc03L306